src/TestCaseRecorder.ts

```diff
@@ -128,32 +96,35 @@ export class TestCaseRecorder {
     }
 
     this.fixtureSubdirectory = subdirectory;
-    return this.promptFileName();
+    return true;
   }
 
-  private async promptFileName(): Promise<string | null> {
+  private calculateFilePath(testCase: TestCase): string {
     if (this.fixtureRoot == null) {
       throw new Error("Missing fixture root. Not in cursorless workspace?");
     }
 
-    const filename = await vscode.window.showInputBox({
-      prompt: "Fixture Filename",
-    });
-
-    if (filename === undefined || this.fixtureSubdirectory == null) {
-      return this.promptSubdirectory(); // go back a prompt
-    }
-
     const targetDirectory = path.join(
       this.fixtureRoot,
-      this.fixtureSubdirectory
+      this.fixtureSubdirectory!
     );
 
     if (!fs.existsSync(targetDirectory)) {
       fs.mkdirSync(targetDirectory);
     }
 
-    this.outPath = path.join(targetDirectory, `${filename}.yml`);
-    return this.outPath;
+    const filename = camelize(testCase.spokenForm);
```

Comment:
Yeah I'd been debating this one. The only problem is that there may be several tests with the same cursorless command. For example, `take funk` is prob what I'd use to test all the different typescript function matchers (eg top-level, class member function, etc). But maybe you have a stop-gap here I've missed?

Reply:
I was thinking that maybe just number them.

Reply:
Maybe simplest is just to append a number if test with that name exists? Prob not worth the trouble of coming up with better test names, tho I guess we could add an option if we think it would be useful

Reply:
Ha ok sounds like we're on the same page 😊

Reply:
Yup!